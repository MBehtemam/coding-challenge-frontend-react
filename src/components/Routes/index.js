import React, { Component } from 'react';
import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';

export default class Routes extends Component {
  render() {
    return (
      <Router>
        <Switch>
          <Route exact path="/" component={() => <span>Main</span>} />
          <Route path="/case/:id" component={() => <span>Detail</span>} />
        </Switch>
      </Router>
    );
  }
}
