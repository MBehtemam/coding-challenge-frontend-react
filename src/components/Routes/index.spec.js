import React from 'react';
import Routes from '.';

describe('Test suits for Routes Component', () => {
  it('should match to snapshot', () => {
    expect(<Routes />).toMatchSnapshot();
  });
});
