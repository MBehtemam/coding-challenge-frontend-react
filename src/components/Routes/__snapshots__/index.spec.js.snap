// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Routes Component should match to snapshot 1`] = `<Routes />`;
